services/productService.js

```diff
@@ -0,0 +1,12 @@
+const productDao = require('../models/productDao');
+
+const getProductByMainpage = async () => {
+  
+  const products = await productDao.getProductByMainpage();
+
+  return products;
+};
```

Comment:
어차피 products를 바로 반환한다면 아래와 같이 작성할 수 있을 것 같습니다.
```suggestion
const getProductByMainpage = async () => {
  return await productDao.getProductByMainpage();
};
```